config/routes.rb

```diff
@@ -1,6 +1,13 @@
 Rails.application.routes.draw do
+
+  # get 'users', controller: 'users', action: 'index'
+  get 'users', to: 'users#index'
+  get 'users/:id', to: 'users#show', as: 'user'
+  get 'users/:user_id/posts', to: 'posts#index', as: 'user_posts'
+  get 'users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
```

Comment:
Great job so far @ikoote1 There are just a few changes I will suggest you make and you'll be on your way to the next Milestone :wink: 
- [ ] The users and posts endpoints appear to be correct and maintain a RESTful URL structure. However, it is important to note that the appropriate HTTP verbs should be used to match the corresponding actions in a RESTful design. Here's an updated version of the routes using the conventional HTTP verbs:

```ruby
resources :users, only: [:index, :show] do
  resources :posts, only: [:index, :show]
end
```
- The code above uses the `resources method`, which generates the conventional RESTful routes for the users and posts resources. The `only` option is used to specify the allowed actions for each resource. :+1: